server/index.js

```diff
@@ -2,22 +2,25 @@ import { ApolloServer } from 'apollo-server-express'
 import { importSchema } from 'graphql-import'
 import express from 'express'
 import path from 'path'
+import redis from 'async-redis'
 import resolvers from './resolvers'
-import { PORT } from './config'
+import { PORT, REDIS_IP, REDIS_PORT } from './config'
 
 const typeDefs = importSchema(path.join(__dirname, './typedefs/index.graphql'))
 
+const cache = redis.createClient({ host: REDIS_IP, port: REDIS_PORT })
+
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  context: ({ req = {}, res }) => ({ req, res }),
+  context: ({ req = {}, res }) => ({ req, res, cache }),
   introspection: true,
   playground: true
 })
 
+cache.on('ready', () => console.log(`🧶  Redis server ready at ${REDIS_IP}:${REDIS_PORT}`))
```

Comment:
Do you need to wait for the cache to be ready before starting the app? What if it is never ready (due to a bad IP address, for example)?